### PR TITLE
Improve animal welfare

### PR DESCRIPTION
**Pull request by malinatran**

### CODE REVIEW

File: knowledge/responses/cant_we_just_improve_the_welfare_of_animals.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
+
+Regardless of the privilege or treatment received by the animals, there is no ethical or humane way to treat animals destined to be exploited and slaughtered. Improving the treatment of animals is better than maintaining the status quo, but it does not provide any substantial relief to the animals themselves and does not address or eradicate the problem of animal abuse and exploitation. Animals don’t want bigger cages or larger barns. All they want is to be free, not “free-range,” but free to live their lives in their entirety, without fear of human-inflicted pain, suffering or exploitation.
+
+### References
+[1] [Alonso ME, González-Montaña JR, Lomillos JM. Consumers' Concerns and Perceptions of Farm Animal Welfare. Animals (Basel). 2020 Feb 27;10(3):385. doi: 10.3390/ani10030385. PMID: 32120935; PMCID: PMC7143148.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7143148/#B11-animals-10-00385)
```
Comment (malinatran) [inline knowledge/responses/cant_we_just_improve_the_welfare_of_animals.md:8]: Is this the correct citation format?

Comment (cvqluu) [reply knowledge/responses/cant_we_just_improve_the_welfare_of_animals.md:8]: Sure, I don't think it's too important to be perfect with references, as long as they're there and have a link

**Comment by cvqluu** [comment]
Looks good to me! Thanks so much!

I've just added a small change to the config file for this response also - now the full comment will be posted.

Thanks again! Approved

**Review by cvqluu — APPROVED** [review]
Approved

**Pull request merged**